tests/scripts/thread-cert/border_router/test_dnssd_server.py

```diff
@@ -221,6 +223,42 @@ def test(self):
                 'status': 'NXDOMAIN',
             })
 
+        # verify Discovery Proxy works for _meshcop._udp
+        dp_service_name = '_meshcop._udp.default.service.arpa.'
```

Comment:
To avoid confusion with the mDNS service published by Border Agent, can we consider using `_ipps._tcp` which is already used for existing DNS-related tests?